packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -223,7 +223,44 @@ class TextStyleTween extends Tween<TextStyle> {
 ///
 /// Which properties are animated is left up to the subclass. Subclasses' States
 /// must extend [ImplicitlyAnimatedWidgetState] and provide a way to visit the
-/// relevant fields to animate.
+/// relevant fields to animate. The "see also" section below lists a number of
+/// concrete subclasses that ship with the framework (their names usually start
+/// with `Animated`).
```

Comment:
Don't they all get listed automatically in the "implements" list?

Reply:
the see also list will always be out of date if it's trying to be comprehensive. maybe it'd be nice to have a section that gives some representative examples and discusses some of the interesting aspects of it, rather than just trying to have the list? We could do things like discuss how AnimatedOpacity uses a special way to minimize dirtiness, for example.